lib/cfg-tree.c

```diff
@@ -1065,6 +1066,38 @@ cfg_tree_compile(CfgTree *self)
   return TRUE;
 }
 
+static gboolean
+are_the_persist_ids_unique(const GPtrArray * const initialized_pipes)
```

Comment:
much better function name :+1: I'd put a `_` before the name as in other places where a function is `static`.